store/iscn.ts

```diff
@@ -118,40 +120,11 @@ export default class ISCN extends VuexModule {
   @Action
   async queryISCNByKeyword(keyword: string): Promise<ISCNRecordWithID[]> {
     this.context.commit('clearRecords');
-    let txRecords: ISCNRecord[] = [];
     let result: ISCNRecord[] = ([] as ISCNRecord[])
     try {
       this.context.commit('setIsLoading', true)
-      const client = await getQueryClient()
-      const [
-        txRes,
-        idRes,
-        fingerprintRes,
-        ownerRes,
-      ] = await Promise.all([
-        client.queryISCNIdsByTx(keyword).catch(() => {}),
-        client.queryRecordsById(keyword).catch(() => {}),
-        client.queryRecordsByFingerprint(keyword).catch(() => {}),
-        client.queryRecordsByOwner(keyword).catch(() => {}),
-      ])
-      if (txRes) {
-        txRecords = (
-          await Promise.all(
-            txRes.map(async (t) => {
-              if (typeof t === 'string') {
-                const res = await client.queryRecordsById(t)
-                return res?.records[0]
-              }
-              return t
-            }),
-          )
-        ).filter((t) => t) as ISCNRecord[]
-      }
-      result = result
-        .concat(txRecords)
-        .concat(idRes ? idRes.records : [])
-        .concat(fingerprintRes ? fingerprintRes.records : [])
-        .concat(ownerRes ? ownerRes.records : [])
+      const { data } = await axios.get(`${network.apiURL}/iscn/records?q=${keyword}`)
```

Comment:
Seems can use axios that provides by the store without extra import
https://axios.nuxtjs.org/usage/#store-actions

Reply:
I have tried `this.$axios`, `$axios`, `this.axios` but none of them are defined. 🤔 There isn't other usage of axios in `store` for reference 🥵

Reply:
would it be package version issue?